README.md

```diff
@@ -9,6 +9,28 @@ With ease:
 - publish page using GitHub-Pages,
 - mirror changes to a separate repository.
 
+## Requirements and Prerequisites
+
+It is necessary to set up the corresponding access rights per Repository for the default `GITHUB_TOKEN`. Please adjust the settings and go to the repository settings -> Actions -> General -> Workflow permissions and specify the read and write access rights for the repository. 
+![General Settings](docs/images/Github_Initial_Git_Push_Settings_General.png)
+![Workflow Settings](docs/images/Github_Initial_Git_Push_Settings_Actions.png)
+
+It is also [possible](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) to set up the appropriate access rights for the GitHub token per workflow. Please follow the attached example to set the correct access rights for the `content` and `pull-requests` scope.
+
+```yaml
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ github.head_ref }}
+        fetch-depth: 0
+```
```

Comment:
```suggestion
The `GITHUB_TOKEN` permissions can also be configured globally for all jobs in a workflow or individually for each job. This example demonstrates how to set the necessary permissions for the `contents` and `pull-requests` scopes on a job level:

```yaml
...

jobs:
  job1:
    runs-on: ubuntu-latest
    permissions:                # Job-level permissions configuration starts here
      contents: write           # 'write' access to repository contents
      pull-requests: write      # 'write' access to pull requests
    steps:
      - uses: actions/checkout@v4
```

To apply permissions globally, which will affect all jobs within the workflow, you would define the `permissions` key at the root level of the workflow file, like so:

```yaml
...

permissions:                    # Global permissions configuration starts here
  contents: read                # 'read' access to repository contents
  pull-requests: write          # 'write' access to pull requests

jobs:
  job1:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
```

Adjust the permission levels and scopes according to your workflow's requirements. For further details on each permission level, consult the [GitHub documentation](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token).

```